test/unit/autoNumeric.spec.js

```diff
@@ -1158,7 +1158,8 @@ describe('The AutoNumeric object', () => {
             aNInput = new AutoNumeric(newInput, 12234678.321, { currencySymbol:'$', formatOnPageLoad: false }); // An initial value, but not formatted on load
             expect(aNInput.getNumericString()).toEqual('12234678.321');
             expect(aNInput.getFormatted()).toEqual('12234678.321');
-            aNInput.node().focus();
+            // aNInput.node().focus();    // simply calling focus() on the node is not enough, because the focus event is fired when the browser window is also active/focused
+            aNInput.node().dispatchEvent(new window.Event('focus', { bubbles: false, cancelable: true }));
```

Comment:
Could you please add the comment on the same line 1162 (and drop the original `// aNInput.node().focus();` ?